src/components/Books.js

```diff
@@ -1,13 +1,23 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import Book from './Book';
 import Form from './Form';
 
 function Books() {
+  const booksData = useSelector((state) => state.bookreducer);
+  console.log(booksData);
```

Comment:
- [ ] (OPTIONAL) Kindly remove all console statements so as to keep your code clean 💯